metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -449,3 +449,15 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: InvalidStorageAutoScalerCRsFound
+      annotations:
+        description: |-
+          Invalid phase detected for StorageAutoScaler {{ $labels.name }} attached with device class {{ $labels.device_class }} under storage cluster {{ $labels.storagecluster }} in the namespace {{ $labels.namespace }}.
+        message: StorageAutoScaler is in an invalid state.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/InvalidStorageAutoScalerCRsFound.md
```

Comment:
@parth-gr , @ShravaniVangur , please note that I've changed the previous name (DuplicateStorageAutoScalerCRsFound.md) to `InvalidStorageAutoScalerCRsFound.md`. We need to make corresponding change to the runbook side.
Or let me know if we have decided on any other name for the file, will make the changes here.

Reply:
Okay thanks

Reply:
Just an FYI: made the change to the alert name, as @parth-gr suggested, `StorageAutoScalerCRIsInvalid` ... So made the same change to the MD file name as well